test/extended/apibinding/apibinding.go

```diff
@@ -0,0 +1,44 @@
+package apibinding
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[sig-apibinding]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-apibinding")
+	)
+
+	g.It("Author:pewang-Critical-[Smoke] Create an apibinding attach a workspace to the exist shared compute should work well", func() {
+		// Share compute could only access from specific test environments
+		// Skip for none supported test environments
+		exutil.PreCheckEnvSupport(k, "kcp-stable.apps.kcp-internal", "kcp-unstable.apps.kcp-internal")
+
+		g.By("# Create an apibinding attach a workspace to the exist shared compute")
+		myAPIBing := NewAPIBinding(SetAPIBindingReferencePath("root:redhat-acm-compute"), SetAPIBindingReferenceExportName("kubernetes"))
+		myAPIBing.Create(k)
+		defer myAPIBing.Delete(k)
+
+		g.By("# Create an deployment should become ready and schedule to the shared compute")
+		myDeployment := exutil.NewDeployment()
+		myDeployment.Create(k)
+		defer myDeployment.Delete(k)
```

Comment:
APIBinding is created within a workspace, Deployment is created within a namespace of a workspace. If workspace will be deleted, so `defer myAPIBing.Delete(k)` and `defer myDeployment.Delete(k)` are not needed.

Reply:
Hmm, yeah it's not must but from my understanding we should clean the resources as more as we can. E.g. for namespace has too many resources if we delete it directly it will keep in Teminating status for a long time not sure the same with worksapce. WDYT?

Reply:
ok. It makes sense for the workloads. But for storage like PV need to use defer to make sure it delete and not cost extra expense. I'll remove the defer for apibinding and deployment.

Reply:
PV is cluster wide scope so it must use `defer` explicitly.